docs/pages/_document.js

```diff
@@ -82,7 +82,7 @@ export default class MyDocument extends Document {
           <link href="https://fonts.gstatic.com" rel="preconnect" crossOrigin="anonymous" />
           <link rel="preconnect" href="https://fonts.googleapis.com" />
           <link
-            href="https://fonts.googleapis.com/css2?family=IBM+Plex+Mono:wght@400;600&display=swap"
+            href="https://fonts.googleapis.com/css2?family=IBM+Plex+Mono:wght@400;600&family=IBM+Plex+Sans:wght@400;500;700&display=swap"
```

Comment:
Do we need multiple font-weight for the code? 400 and 600?

Reply:
Is 400 not too crisp

Reply:
[400 seems to be the regular weight on this typeface](https://fonts.google.com/specimen/IBM+Plex+Sans?query=ibm). So the scale goes:

- 400 regular
- 500 medium
- 600 semibold
- 700 bold

If we remove the 400 one, we'd be setting the medium weight as default, which would mess up a bit the design 😅

Reply:
1. 500

<img width="465" alt="Capture d’écran 2021-08-04 à 22 52 07" src="https://user-images.githubusercontent.com/3165635/128253395-98985dfd-af57-4f3a-babc-5132ad1ff126.png">

2. 400

<img width="468" alt="Capture d’écran 2021-08-04 à 22 52 22" src="https://user-images.githubusercontent.com/3165635/128253521-5c30c2dc-7dda-46ae-b859-c03f7e108b22.png">

3. 400 with subpixel-antialiased.

<img width="603" alt="Capture d’écran 2021-08-04 à 22 53 41" src="https://user-images.githubusercontent.com/3165635/128253900-5d0efbc6-a6b0-461f-a6b7-587868e5dfc6.png">

I would recommend 3, the option we already used in the docs.

Reply:
Ah, sorry. I didn't notice you were talking about the code typeface. Agree with you on the third option tho.

Reply:
will add this point to improvement phase, so far I don't have issue with `400`